machine.md

```diff
@@ -252,6 +252,33 @@ spec:
       persistentVolumeClaim:
         claimname: myclaim
 ```
+## Resources Requests and Limits
+
+An optional Resources requests can be specified by the users to
+allow the scheduler to make a better decision in finding the most suitable Node
+to place the Virtual Machine on.
+
+```
+    resources:
+      requests:
+        memory: "1Gi"
+        cpu: "2"
+      limits:
+        memory: "2Gi"
+        cpu: "1"
+```
+
+#### CPU
+Specifying CPU limits will determine the amount of time Virtual Machine CPUs
```

Comment:
It would be good if `cpu shares` can be sneaked in to make it clear what is assigned to the VM.

Maybe look at the cgroups or pod resource documentation for ideas.